docs/sphinx/userguide/gpioz.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../../../src/jukebox/components/gpio/gpioz/README.rst
```

Comment:
I wonder why we put this docs into the code rather than in the docs folder. I am not saying this is a bad idea, I actually like it, but should we start doing this from now on? What's your thought?

Reply:
I have put it this way
- README.rst: The User Guide that goes into the RTD User Guide Section
- Python inline code: The API Documentation that goes into the RTD API Section. 
Both in one file is difficult, as they both go to different section in the Docs.

Is certrainly not consistent across the project. README.rst is also not the perfect name. USERGUIDE.rst would be more accurate. But READEME.rst is automatically shown on Github. USERGUIDE.rst - don't know.